templates/skeleton/server.ts

```diff
@@ -75,6 +84,11 @@ export default {
 
       const response = await handleRequest(request);
 
+      response.headers.set(
+        'Content-Security-Policy',
+        `base-uri 'self'; default-src 'self' 'nonce-${nonce}' https://cdn.shopify.com; frame-ancestors 'none'; style-src 'self' 'unsafe-inline' https://cdn.shopify.com;`,
```

Comment:
If we do abstract something for CSP, this line will still need to remain very customizable.